lib/Weathermap/UI/UIBase.php

```diff
@@ -98,7 +100,7 @@ public function validateArgument($type, $value)
         }
 
         if ($type != "") {
-            throw new WeathermapInternalFail("ValidateArgs saw unknown type");
+            throw new WeathermapInternalFail("ValidateArgs saw unknown type '$type'");
```

Comment:
I'm not sure about this line, namely the addition of single quotes

Reply:
With the single quotes you can see the difference between an empty value, a value with spaces and typos in the message.

However, I'll remove it. :-)

Reply:
I would leave it as is.